internal/inspect/inspect_test.go

```diff
@@ -30,14 +30,15 @@ import (
 
 func TestInspect(t *testing.T) {
 	type policies struct {
-		expected               map[string]*responsev1.InspectPoliciesResponse_Result
-		expectedMissingImports []string
+		expected map[string]*responsev1.InspectPoliciesResponse_Result
```

Comment:
I think these tests are getting quite complicated and hard to follow. For example, the helper functions like `attributes` and `result` have so many parameters and you need to keep looking at their signatures to figure out what each one is. These  tests are a good candidate for moving into file-based tests like we do with engine, compile and server tests.